Revise WebGPU status and feature support details - Status page

thomlucc opened the pull request:
Updated the WebGPU status documentation to reflect the current implementation status and optimizations.

deltakosh commented on content/How_To/material/Reflect.md:
```diff
@@ -5,6 +5,20 @@ PG_TITLE: 04. How To Obtain Reflections and Refractions
 # How To Obtain Reflections and Refractions
 Using reflection textures can simulate mirror like material and refraction textures can simulate looking through glass or water.
 
+# Table of contents
```
The TOC is built automatically no need to add it

ddervisis replied:
TOC removed

deltakosh commented on content/How_To/material/Reflect.md:
```diff
@@ -81,6 +95,27 @@ skyboxMaterial.reflectionTexture = new BABYLON.HDRCubeTexture("PATH TO HDR IMAGE
 
 * [Playground Example of HDR Skybox](http://www.babylonjs-playground.com/#114YPX#5)
 
+### EquirectangularCubeTexture
```
```suggestion
### EquiRectangularCubeTexture
```

ddervisis commented:
I just updated the Babylon-Playground examples.

deltakosh commented on content/How_To/material/Reflect.md:
```diff
@@ -81,6 +81,28 @@ skyboxMaterial.reflectionTexture = new BABYLON.HDRCubeTexture("PATH TO HDR IMAGE
 * [Playground Example of HDR Skybox](http://www.babylonjs-playground.com/#114YPX#5)
 
+### EquiRectangularCubeTexture
+Equirectangular images are browser-canvas supported images like jpeg, png, and many more. A list of image support on browsers can be found [here](https://en.wikipedia.org/wiki/Comparison_of_web_browsers#Image_format_support).
```
```suggestion
EquiRectangular images are browser-canvas supported images like jpeg, png, and many more. A list of image support on browsers can be found [here](https://en.wikipedia.org/wiki/Comparison_of_web_browsers#Image_format_support).
```

deltakosh commented:
Thanks!

thomlucc committed:
Revise WebGPU status and feature support details

Updated the WebGPU status documentation to reflect the current implementation status and optimizations.

bjsplat commented:
snapshot for this PR available at:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/DOCS/refs/pull/1432/merge
